ONSdigital/dp-frontend-renderer
# Feature/add loop11 flag

### What

Removed the loop11 partial and re-formatted each individual template file that included it, and replaced it with a conditional call located in the `main.tmpl` file. This allows us to enable/disable loop11 for specific controllers via a flag (default set to `false`).

### How to review

1. Pull `feature/add-loop11-flag` branch for `dp-frontend-renderer`, `dp-frontend-models`, `dp-frontend-dataset-controller`, `dp-frontend-filter-dataset-controller` and `dp-geography-controller`.
2. Enable loop11 for each controller by including the flag, `ENABLE_LOOP11=true`
3. Check in network tab that loop11.js is present when visiting the specific pages for that controller.
4. Sense check that the loop11 script isn't loaded when the controller is run without the flag.

### Who can review

Anyone
